changelog.d/opentelemetry_trace.feature.md

```diff
@@ -0,0 +1,4 @@
+Support ingesting opentelemetry traces(Experimentally)
+
+authors: caibirdme
+
```

Comment:
```suggestion
```

The CI job is unable to find the authors because it is looking in the last line of the file for it.